tests/e2e/env/package.json

```diff
@@ -41,6 +41,10 @@
     "access": "public"
   },
   "scripts": {
+    "clean": "rm -rf ./build ./build-module",
+    "compile": "node ./../bin/build.js",
```

Comment:
Should this be moved to the main repo package.json. The build script won't exist in the `/node_modules/` folder when installed as a package.

Reply:
I feel like that's okay because we shouldn't be running any of these scripts from a parent project right? The build script is for preparing the packages for distribution so they aren't really relevant once distributed? The incentive for having it here is that we can run `lerna run build` and then each package in the monorepo is able to define its own build script as opposed to having a single universal script.